pkg/runtime/runtime.go

```diff
@@ -1794,20 +1739,14 @@ func (a *DaprRuntime) initState(s componentsV1alpha1.Component) error {
 		// when placement address list is not empty, set specified actor store.
 		if len(a.runtimeConfig.PlacementAddresses) != 0 {
 			// set specified actor store if "actorStateStore" is true in the spec.
-			actorStoreSpecified := false
-			for k, v := range props {
-				if strings.ToLower(k) == actorStateStore { //nolint:gocritic
-					actorStoreSpecified = utils.IsTruthy(v)
-				}
-			}
-
-			if actorStoreSpecified {
+			actorStoreSpecified := props[actorStateStore]
+			if actorStoreSpecified == "true" {
```

Comment:
Please revert this change too, which was made on purpose to make `actorstatestore` case-insensitive and match with IsTruthy